signature/types.go

```diff
@@ -145,3 +145,18 @@ func (signerInfo *SignerInfo) ExtendedAttribute(key string) (Attribute, error) {
 	}
 	return Attribute{}, errors.New("key not in ExtendedAttributes")
 }
+
+// GetAuthenticSigningTime returns the authentic signing time
+func (signerInfo *SignerInfo) GetAuthenticSigningTime() (time.Time, error) {
```

Comment:
According to [effective go](https://go.dev/doc/effective_go#Getters), it is better to have
```suggestion
func (signerInfo *SignerInfo) AuthenticSigningTime() (time.Time, error) {
```
so that it will be also consistent with the above `func (signerInfo *SignerInfo) ExtendedAttribute(key string) (Attribute, error)`.